src/github.com/getlantern/balancer/dialer.go

```diff
@@ -131,7 +131,7 @@ func (d *dialer) defaultCheck() bool {
 			Dial: d.Dial,
 		},
 	}
-	ok, timedOut, _ := withtimeout.Do(10*time.Second, func() (interface{}, error) {
+	ok, timedOut, _ := withtimeout.Do(60*time.Second, func() (interface{}, error) {
```

Comment:
Thanks @myleshorton. Otherwise the PR looks good to me, let's keep going and discuss this cosmetic detail any other day.